k8s/demo/common/dg-docassembly/dg-docassembly-demo.yaml

```diff
@@ -0,0 +1,50 @@
+---
+apiVersion: flux.weave.works/v1beta1
+kind: HelmRelease
+metadata:
+  name: dg-docassembly-demo
+  namespace: dg-docassembly
+  annotations:
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.java: glob:prod-*
+spec:
+  releaseName: dg-docassembly-demo
+  timeout: 900
+  forceUpgrade: true
+  rollback:
+    enable: true
+    force: true
+  chart:
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    name: dg-docassembly
+    version: 0.0.1
```

Comment:
is this correct? 🤔 just think i remember a bigger num

Reply:
Not sure. Just started with this number as this was the first version of the file